presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -1306,6 +1306,34 @@ public void testRemoveRedundantCrossJoin()
                                         values(ImmutableMap.of())))));
     }
 
+    @Test
+    public void testRemoveRedundantInnerJoin()
+    {
+        assertPlan("SELECT regionkey FROM nation INNER JOIN (SELECT nationkey FROM customer LIMIT 0) using(nationkey)",
```

Comment:
Capitalize `USING` and add a space between it and the parenthesis. I.e., `USING (nationkey)`